docs/build/arm64-exception-handling.md

```diff
@@ -313,6 +313,7 @@ The unwind codes are encoded according to the table below. All unwind codes are
 |  | 11101010: Custom stack for MSFT_OP_CONTEXT |
 |  | 11101100: Custom stack for MSFT_OP_CLEAR_UNWOUND_TO_CALL |
 |  | 1111xxxx: reserved |
+| `pac_sign_return_address` | 11111100: sign the return address in `x30` with `pacibsp` |
```

Comment:
`x30` is referred to as `lr` everywhere else in this document. We should use `lr` here was well for consistency.

Reply:
Thanks - I updated the branch, fixing this issue now, so that it references `lr` instead.